coreos-base/oem-packet/oem-packet-0.0.5.ebuild

```diff
@@ -16,14 +16,11 @@ S="${WORKDIR}"
 
 src_prepare() {
 	sed -e "s\\@@OEM_VERSION_ID@@\\${PVR}\\g" \
-	    "${FILESDIR}/cloud-config.yml" > "${T}/cloud-config.yml" || die
+		"${FILESDIR}/oem-release" > "${T}/oem-release" || die
 }
 
 src_install() {
-	into "/usr/share/oem"
-	dobin "${FILESDIR}/phone-home.sh"
```

Comment:
Did you push the branch?  I don't see a difference here.  The service still doesn't appear to get installed anywhere.